hknweb/tours/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from . import views
+
+app_name = 'tours'
+urlpatterns = [
+    path('', views.tour, name='index'),
+    # path('confirm', views.confirm, name='confirmtour'),
```

Comment:
Let's remove this line if it's not being used? If it's there as a placeholder, let's open an issue and mark it with a more detailed comment in the code or smth?